src/logging.ts

```diff
@@ -252,10 +252,17 @@ export class Logger {
         SingletonLogger.instance().clearOutputChannel();
     }
 
-    showChannel() {
+    showChannel(target_failed?: boolean) {
```

Comment:
```suggestion
    showChannel(errorToShow?: boolean) {
```

Reply:
I think modifiying this makes it a little more generic about the error rather than "target_failed", somewhat of a nit change, but I think for future development it could help.

Reply:
i used snake case instead for [fe04bb8](https://github.com/microsoft/vscode-cmake-tools/pull/3785/commits/fe04bb86d560ff7cd53e55066f7c2ea3c185beaa)